cypress/e2e/api/GetPostById.spec.js

```diff
@@ -0,0 +1,41 @@
+describe("Posts", () => {
+  it('should return a post by ID', () => {
+    const post = {
+      userId: 1,
+      id: 1,
+      title: "sunt aut facere repellat provident occaecati excepturi optio reprehenderit",
+      body: "quia et suscipit\nsuscipit recusandae consequuntur expedita et cum\nreprehenderit molestiae ut ut quas totam\nnostrum rerum est autem sunt rem eveniet architecto"
+    };
+
+
+    cy.api({
+      url: `/posts/${post.id}`,
+      method: 'GET',
+    }).as("getPost");
+
+    cy.get("@getPost").its("status")
```

Comment:
u can compare whole object but not properties one by one 
![image](https://github.com/AlinaDashko/hillel-ngx-cypress-test/assets/53534240/68416ae9-2c2d-49cd-9e8c-aab1ebaff08d)